js/main.js

```diff
@@ -0,0 +1,156 @@
+
+const canvas = document.getElementById('wordSearchCanvas');
+const context = canvas.getContext('2d');
+
+
+function redraw(){
+
+  var height = canvas.getBoundingClientRect().height;
+  var width = canvas.getBoundingClientRect().width;
+
+  //console.log('Height is: ' + height + ' Width is: ' + width);
+}
+
+/**
+ * Returns a word seach game board that is created based of the dimensions
+ * of the allocated screen space with each letter being allocated it's
+ * own square within the board.
+ * @param {number} canvasHeight 
+ * @param {number} canvasWidth 
+ * @returns {Object}
+ */
+function createGameBoard(canvasHeight, canvasWidth){
+  var gameBoard = {
```

Comment:
There's rarely a reason to use `var` instead of `const` or `let`.

Google style is to use `const` if possible; if not `const`, then `let`; and only then `var` if you really have to.

Reply:
I actually did not know really understand the diff between var and let until now. Updated accordingly.